sonic_installer/main.py

```diff
@@ -591,7 +591,18 @@ def install(url, force, skip_platform_check=False, skip_migration=False, skip_pa
 
         echo_and_log("Installing image {} and setting it as default...".format(binary_image_version))
         with SWAPAllocator(not skip_setup_swap, swap_mem_size, total_mem_threshold, available_mem_threshold):
-            bootloader.install_image(image_path)
+            try:
+                bootloader.install_image(image_path)
+            except BaseException as e:
```

Comment:
>BaseException [](http://example.com/codeflow?start=19&length=13)

Can we use more specific exception type relating disk issue?

Reply:
Fixed, install image failed with throw SystemExit exception.